src/AspNet.Security.OAuth.Introspection/OAuthIntrospectionHandler.cs

```diff
@@ -31,12 +31,12 @@ public class OAuthIntrospectionHandler : AuthenticationHandler<OAuthIntrospectio
 
             // Ensure that the authorization header contains the mandatory "Bearer" scheme.
             // See https://tools.ietf.org/html/rfc6750#section-2.1
-            if (!header.StartsWith("Bearer ", StringComparison.OrdinalIgnoreCase)) {
+            if (!header.StartsWith($"{OAuthIntrospectionDefaults.AuthenticationScheme} ", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
TBH, I'm not a huge fan of this change, because despite its name, `AuthenticationScheme` is really meant to be an internal identifier, not a public thingy.

If you're **really** convinced being able to replace the "scheme" is worth it, I'd suggest creating a separate property. Note that using another scheme would make this middleware non-standard: https://tools.ietf.org/html/rfc6750#section-2.1